### PR TITLE
Unify documentation code formatting

### PR DESCRIPTION
**Pull request by mcktr**
## Current Behavior
There are misformatted code statements in our documentation. 

https://icinga.com/docs/icinga2/latest/doc/17-language-reference/#apply-for
auswahl_006

Notice the `=&gt;`.

https://icinga.com/docs/icinga2/latest/doc/05-service-monitoring/#create-a-new-plugin
auswahl_008

Notice `&lt;` and `&gt;`. 

This happens at multiple locations and is only visible in the rendered documentation on https://icinga.com/docs/icinga2/latest/.

## Steps to Reproduce (for bugs)
I can reproduce this locally with the MkDocs extension `codehilite`.

```diff
diff --git a/mkdocs.yml b/mkdocs.yml
index 8fa7bcfec..f77b6d422 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -29,5 +29,7 @@ pages:
 theme: readthedocs
 markdown_extensions:
   - smarty
+  - codehilite:
+      guess_lang: true
 extra_javascript:
   - scroll.js
```

```
mkdocs serve
```

## Context
The extension `codehilite` provides syntax highlighting for code blocks and is used in the [icinga-docs-tools](https://github.com/Icinga/icinga-docs-tools).

## Possible Solution

I investigated on the issue and found out, that this only happens in code blocks formatted with four spaces. Code blocks that are formatted with back-ticks are not affected by this issue.

This PR unifies the documentation code formatting to only use back-ticks.

### CODE REVIEW

File: doc/13-addons.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ You need to install Graphite first, then proceed with configuring it in Icinga 2
 Use the [GraphiteWriter](14-features.md#graphite-carbon-cache-writer) feature
 for sending real-time metrics from Icinga 2 to Graphite.
 
-    # icinga2 feature enable graphite
+```
+# icinga2 feature enable graphite
```
Comment (dnsmichi) [inline doc/13-addons.md]: The leading `#` causes rendering problems when a list is put beforehand (on GH or with mkdocs, I do not remember atm). We should decide whether to drop the root shell character here.

Comment (mcktr) [reply doc/13-addons.md]: How about the following:

```
$ sudo icinga2 feature enable graphite
```

For example Graylog uses this to indicate that something needs to be run as root.

Comment (dnsmichi) [reply doc/13-addons.md]: I would entirely drop it, since there's also the possibility to run these commands on Windows cmd/PS.

Comment (mcktr) [reply doc/13-addons.md]: Agreed. I will push a new commit to update this PR. Should I remove the leading `#` only in `13-addons.md` or in the entire documentation?

Comment (dnsmichi) [reply doc/13-addons.md]: Mh. Leave it as is for now, and we'll deal with this change in a different PR. If you feel sane about this PR, feel free to merge.

**Pull request merged**